packages/server/src/unstable-core-do-not-import/stream/utils/promiseTimer.ts

```diff
@@ -1,40 +1,24 @@
-import { createDeferred } from './createDeferred';
+// @ts-expect-error polyfill
+Symbol.dispose ??= Symbol();
```

Comment:
Should probably be moved to some "globals" file or something...

Reply:
no guarantee it's imported if it's in a global

here it's guaranteed to be imported in relation to this

i'm trying to get the real thing working in #6217 if you feel like poking